### PR TITLE
Make `lora` weights repos compatible with the text2image task on `diffusers`

### PR DESCRIPTION
**Pull request by apolinario**

### CODE REVIEW

File: docker_images/diffusers/app/pipelines/text_to_image.py [hunk 1]
```diff
@@ -18,6 +19,14 @@
 class TextToImagePipeline(Pipeline):
     def __init__(self, model_id: str):
 
+        model_data = model_info(model_id, token=os.getenv("HF_API_TOKEN"))
+        is_lora = any(file.rfilename == 'pytorch_lora_weights.bin' for file in model_data.siblings)
```
Comment (Narsil) [inline docker_images/diffusers/app/pipelines/text_to_image.py:23]: No `safetensors`  version of it ?  :( :D

Comment (apolinario) [reply docker_images/diffusers/app/pipelines/text_to_image.py]: Great point, I think our training scripts don't yet save `safetensors` lora weights automatically. But I think could be a nice addition! cc @patrickvonplaten @patil-suraj

**Review by Narsil — APPROVED** [review]
LGTM.

Any model I could try out ?

**Comment by apolinario** [comment]
Yes: https://huggingface.co/sayakpaul/sd-model-finetuned-lora-t4

File: docker_images/diffusers/app/pipelines/text_to_image.py (hunk 1 above)
Comment (patil-suraj) [reply docker_images/diffusers/app/pipelines/text_to_image.py]: Good point @apolinario , we could support saving `safetensors` in training scripts.

**Pull request merged**